CHANGELOG.md

```diff
@@ -0,0 +1,27 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+---
+## [master] - TBR
+* Support Managing Environments
+* Support Authorizing Application Access Grants in Terraform.
+
+## [1.0.3] - 2023-04-19
+* Documentation update describing that Stream has been renamed to Topic in Self-Service UI.
+
+## [1.0.2] - 2022-10-13
+* Documentation update. Moved guides into separate folder for guides. Fixed the version to latest in documentation examples.
+
+## [1.0.1] - 2022-10-11
+* Documentation update on using Terraform Provider with Axual Trial
+
+## [1.0.0] - 2022-10-08
```

Comment:
the square brackets are placeholder for link, maybe we can have compare tags link when pressing on a tag link